### ISSUE TITLE
DeprecationWarning:  .ix is deprecated. Please use .loc or .iloc for indexing

### ISSUE THREAD

**Issue by ooduor** [bmabey/pyLDAvis#116]
DeprecationWarning: .ix is deprecated. Please use .loc for label based indexing or .iloc for positional indexing

```
import pyLDAvis
import pyLDAvis.sklearn
pyLDAvis.enable_notebook()
pyLDAvis.sklearn.prepare(lda, tf, tf_vectorizer)
```

Full Jupyter error log.

```
/home/anthony/anaconda3/lib/python3.6/site-packages/pyLDAvis/_prepare.py:387: DeprecationWarning: 
.ix is deprecated. Please use
.loc for label based indexing or
.iloc for positional indexing

See the documentation here:
http://pandas.pydata.org/pandas-docs/stable/indexing.html#deprecate_ix
  topic_term_dists = topic_term_dists.ix[topic_order]
```

Python 3.6.1 |Anaconda custom (32-bit)| (default, May 11 2017, 13:08:54) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-1)]

**Comment by bmabey** [comment]
Are you using ` v2.1.2`?

**Comment by neowangkkk** [comment]
Have you had answer on this? I met the same problem.

**Comment by bmabey** [comment]
Can you tell me what version you are using?

**Comment by neowangkkk** [comment]
Python 3.6.3 :: Anaconda custom (64-bit)

Thank you very much.

**Comment by bmabey** [comment]
What version of pyLDAvis?

**Comment by bmabey** [comment]
Try upgrading to `2.1.2` to see if it fixes the problem.

**Comment by neowangkkk** [comment]
It works great! Thank you very much Ben!
